husky_extensions/ckeditor-extension.js

```diff
@@ -140,20 +172,36 @@
                         CKEDITOR.plugins.add(name, plugin);
                     },
 
-                    addToolbarButton: function(toolbarName, button, icon) {
-                        if (!toolbar[toolbarName]) {
-                            toolbar[toolbarName] = [];
+                    addToolbarButton: function(toolbarName, button, icon, buttonRelations) {
+                        if (!availableToolbarSections[toolbarName]) {
+                            availableToolbarSections[toolbarName] = [];
                         }
 
-                        toolbar[toolbarName].push(button);
+                        availableToolbarSections[toolbarName].push(button);
 
                         if (!!icon) {
                             icons[button] = icon;
                         }
+
+                        if (!!relations) {
```

Comment:
What is this check for? This variable always exists, doesn't it? And if there would be the possibility that it doesn't exist, just ignoring is probably not the best solution, is it?